centreon/src/Centreon/Application/Controller/PlatformController.php

```diff
@@ -78,7 +78,8 @@ function ($version) {
                         return $this->extractVersion($version);
                     },
                     $widgetsVersion
-                )
+                ),
+                'is_cloud_platform' => $_SERVER['IS_CLOUD_PLATFORM'] || false
```

Comment:
It should be injected through Dependency Injection from configuration instead of using global $_SERVER. I can show you how to do that